tractolearn/learning/Datasets.py

```diff
@@ -0,0 +1,452 @@
+# -*- coding: utf-8 -*-
```

Comment:
Same comment as above concerning the module name. Also use singular nouns. Different classes may need to go into separate files.